Inc/DirectXMathMisc.inl

```diff
@@ -2026,8 +2026,8 @@ inline bool XMVerifyCPUSupport()
         return false; // No SSE2/SSE support
 
 #if defined(__AVX2__) || defined(_XM_AVX2_INTRINSICS_)
-#ifdef __clang__
-    __cpuid_count(0, 7, CPUInfo[0], CPUInfo[1], CPUInfo[2], CPUInfo[3]);
+#if defined(__clang__) || defined(__GNUC__) 
+    __cpuid_count(7, 0, CPUInfo[0], CPUInfo[1], CPUInfo[2], CPUInfo[3]);
```

Comment:
Good catch. Fixed in [this commit](https://github.com/microsoft/DirectXMath/commit/549b51d9cc328903739d5fbe7f348aec00c83339)